OSOE-403: ⭐Migrate away from System.Drawing and update Microsoft.SqlServer.DacFx in Lombiq.UITestingToolbox

sarahelsaig opened the pull request:
OSOE-403

Note: If UI test fails because "RequestFailedException: The value for one of the HTTP headers is not in the correct format." update Azurite to the latest version.

sarahelsaig committed:
Remove all "using System.Drawing" and replace them with "using SixLabors.ImageSharp".

sarahelsaig committed:
Update rest of code.

sarahelsaig committed:
Update Microsoft.SqlServer.SqlManagementObjects because it's not forwards compatible.

sarahelsaig committed:
Trust server certificate.

sarahelsaig committed:
Disable Encrypt option in connection string template.

See https://github.com/dotnet/SqlClient/blob/main/release-notes/4.0/4.0.0.md#breaking-changes

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-403

# Conflicts:
#	test/Lombiq.UITestingToolbox

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-403

# Conflicts:
#	test/Lombiq.UITestingToolbox

sarahelsaig committed:
Remove System.Drawing.Common reference again.

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-403

sarahelsaig committed:
Add MultiSubnetFailover as well.

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-403

# Conflicts:
#	src/Modules/Lombiq.ChartJs
#	test/Lombiq.UITestingToolbox

sarahelsaig committed:
Update actions too.

Piedone commented:
On the next commit, UI testing will break if you don't merge from `dev` beforehand.

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-403

# Conflicts:
#	tools/Lombiq.GitHub.Actions

sarahelsaig committed:
spelling

sarahelsaig committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-403

# Conflicts:
#	test/Lombiq.UITestingToolbox
#	tools/Lombiq.GitHub.Actions

sarahelsaig committed:
Guess you don't need MultiSubnetFailover=True (on Windows) after all.

sarahelsaig committed:
Prevent NRE when the file doesn't exist.

sarahelsaig committed:
unusing

dministro committed:
Targeting dev in workflow
Updating submodules:
 - Lombiq.ChartJs: fast-forward
 - Lombiq.UITestingToolbox: fast-forward
 - Lombiq.GitHub.Actions: fast-forward

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-403
